scripts/drivers/_custom.sh

```diff
@@ -42,6 +42,7 @@ driver_decrypt_file() {
         _SECRET="${EXPRESSION#* }"
 
         if ! SECRET=$(_custom_driver_get_secret "${type}" "${_SECRET}"); then
+            rm -f "${output_yaml}" "${output_yaml_anchors}"
```

Comment:
I would recommend to use trap function here.

Reply:
There is already a trap setup elsewhere, when I use a new trap it overwrite the previous one.